utide/periodogram.py

```diff
@@ -65,7 +65,8 @@ def fbndavg(P, freq, cfreq=None, fbands=None):
     if fbands is None:
         fbands = freq_bands
 
-    df = freq[2] - freq[1]
+    # FIXME: Assigned to but never used.
```

Comment:
Fix it now--just delete the line with the assignment.  Same below.  And you may delete the commented-out code lines.